framework/DataObjects/HistorySet.py

```diff
@@ -254,6 +278,29 @@ def _updateSpecializedOutputValue(self,name,value,options=None):
       @ In, value, ?, ?
       @ Out, None
     """
+
+    ## Check if we need to reduce the dataset
+    value = np.atleast_1d(value).flatten()
+    rows = None
+    if self._dataParameters is not None:
+      rows = self._dataParameters.get('outputRow', None)
```

Comment:
For the HistorySet, what is the meaning of the "outputRow" or "operator"?

Reply:
If you use them (and from the gold file I see that) you are going to get the equivalent of a PointSet.

Reply:
Is this not what the user would expect?

I may have been the wrong person to edit this feature. I am not sure I understand the use case here.